specs/hotel_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'spec_helper'
+
+describe "Hotel module" do
+
+  # describe "max room constant" do
+  #
+  #   it "sets the maximum number of rooms in hotel to 20" do
+  #     admin = Hotel::Admin.new
+  #
+  #     admin.MAX_NUM_ROOMS.must_equal 20
```

Comment:
In general it's not helpful to test static behavior. We know it's 20, it's set to 20 - there's no logic to test here.